### ISSUE TITLE
Creator fails after setting survey locale in private mode

### ISSUE THREAD

**Issue by juzraai** [surveyjs/survey-creator#1070]
### Are you requesting a feature, reporting a bug or ask a question?

Reporting a bug.

### What is the current behavior?

After I change to a non-default locale in survey settings, I cannot edit my survey anymore:

- Properties panel turns empty,
- questions disappear when I add/move them,
- survey not being saved,
- JS errors appear in Console on every operation (question select/add/move).

(Detailed below at reproducing steps.)

### What is the expected behavior?

I expect to be able to edit my survey regardless of survey locale settings.

### How would you reproduce the current behavior (if this is a bug)?

1. Open https://surveyjs.io/create-survey in Chrome incognito mode.
2. Add a Single Input question to the survey.
3. Click on Survey Settings.
4. Modify the locale from "Default (English)" to anything else. (I wanted to use "magyar" (Hungarian), but tried others too for this report.)
5. Click on your previously added Single Input -> Properties panel is now empty. Error 1 appears in Console.
6. Add another Single Input -> It disappears. Error 2 appears in Console.

Error 1:

```
Uncaught TypeError: Cannot read property 'join' of undefined
    at LocalizableStrings.get [as text] (localizablestring.ts:259)
    at JsonObjectProperty.getPropertyValue (jsonobject.ts:240)
    at SurveyPropertyStringsEditor../src/propertyEditors/propertyEditorBase.ts.SurveyPropertyEditorBase.getValue (propertyEditorBase.ts:440)
    at SurveyPropertyStringsEditor../src/propertyEditors/propertyEditorBase.ts.SurveyPropertyEditorBase.updateValue (propertyEditorBase.ts:426)
    at SurveyPropertyStringsEditor../src/propertyEditors/propertyEditorBase.ts.SurveyPropertyEditorBase.setObjectCore (propertyEditorBase.ts:262)
    at SurveyPropertyStringsEditor.set [as object] (propertyEditorBase.ts:228)
    at SurveyObjectProperty.set [as object] (objectProperty.ts:74)
    at SurveyElementEditorTabModel../src/questionEditors/questionEditor.ts.SurveyElementEditorTabModel.createEditor (questionEditor.ts:488)
    at SurveyElementEditorTabModel../src/questionEditors/questionEditor.ts.SurveyElementEditorTabModel.buildEditorProperties (questionEditor.ts:483)
    at new SurveyElementEditorTabModel (questionEditor.ts:395)
```

Error 2:

```
Uncaught TypeError: Cannot read property 'join' of undefined
    at LocalizableStrings.get [as text] (survey.ko.js:6939)
    at JsonObjectProperty.getPropertyValue (survey.ko.js:347)
    at SurveyPropertyStringsEditor../src/propertyEditors/propertyEditorBase.ts.SurveyPropertyEditorBase.getValue (survey-creator.js:28094)
    at SurveyPropertyStringsEditor../src/propertyEditors/propertyEditorBase.ts.SurveyPropertyEditorBase.updateValue (survey-creator.js:28079)
    at SurveyPropertyStringsEditor../src/propertyEditors/propertyEditorBase.ts.SurveyPropertyEditorBase.setObjectCore (survey-creator.js:27909)
    at SurveyPropertyStringsEditor.set [as object] (survey-creator.js:27862)
    at SurveyObjectProperty.set [as object] (survey-creator.js:22759)
    at SurveyElementEditorTabModel../src/questionEditors/questionEditor.ts.SurveyElementEditorTabModel.createEditor (survey-creator.js:31571)
    at SurveyElementEditorTabModel../src/questionEditors/questionEditor.ts.SurveyElementEditorTabModel.buildEditorProperties (survey-creator.js:31566)
    at new SurveyElementEditorTabModel (survey-creator.js:31451)
```

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: https://surveyjs.io/create-survey

### Specify your

* browser:
  * Google Chrome 88.0.4324.104 (also 87.0.4280.141)
  * Microsoft Edge 88.0.705.50
* editor version: the one hosted on https://surveyjs.io/create-survey

**Comment by andrewtelnov** [comment]
@juzraai Fixed by the commit above. The fix will be available in v1.8.27 that we will likely release tomorrow.

Thank you,
Andrew

**Comment by juzraai** [comment]
Thank you, it is working now indeed! :)
